### ISSUE TITLE
torch.nn.BCELoss()

### ISSUE THREAD

**Issue by Fishhao123** [GajuuzZ/Human-Falling-Detect-Tracks#21]
Hi! Awesome work！However I found you use BCELoss as your loss in train.py? Can you explain why BCE but not CrossEntropyLoss？ Thanks.

**Comment by GajuuzZ** [comment]
Because I use Smooth Labeling to label each frame. Think about when you are standing up your action maybe Sitting: 50%(decreasing) and Standing: 50%(increasing) on each frame. And use means of each label over the input frames (30) to be the target of an input. So I use Sigmoid as the last activation and use BCELoss for the model will be able to predict the probability of each Action independently. (The label no need to have a sum equal to 1 as if to use Softmax and CrossEntropyLoss).

**Comment by Fishhao123** [comment]
Oh I found your sigmoid function in your model. Have you try the performance difference between using sigmoid or softmax? One more thing, I read the original code of ST-GCN, it doesn't use any activation function as last layer (i.e., the last layer is fcn).

**Comment by Fishhao123** [comment]
I guess I know the reason why ST-GCN didn't use softmax output directly. Cause in Pytorch crossentropy loss first contains softmax layer. Any way thank you very much.
